File: Dotnet/Friday.RockPaperScissors/src/Friday.RockPaperScissors.Tests/RockPaperScissorsTest.cs
using NUnit.Framework;

namespace Friday.RockPaperScissors.Tests;

public class RockPaperScissorsTest
{
    [Test]
    public void RockBeatsScissors()
    {
        var input = new[] {"rock paper"};
        var game = new RockPaperScissors();
        var result = game.Play(input);
        
        Assert.Equals(result, "Player 2 wins: paper beats rock");
    }
}

File: Java/src/test/java/RockPaperScissorsTest.java
import io.github.joaorbrandao.RockPaperScissors;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class RockPaperScissorsTest {
    @Test
    void rockBeatsScissors() {
        var game = new RockPaperScissors("rock paper");
        assertEquals("Player 2 wins: paper beats rock", game.play());
    }
}
